### PR TITLE
feat: Support for providing Platform token

### PR DESCRIPTION
**Pull request by bsenth200**
## What

What does this PR add or remove?

## Why

Why are these changes needed?

## How

How do these changes achieve the goal?

## Test

How has this been tested? How can a reviewer test it?

## Checklist

- [x] I have self-reviewed this PR
- [ ] I have added tests that prove the feature works or the fix is effective

### CODE REVIEW

File: core/sdk/src/framework/ripple_contract.rs [hunk 1]
```diff
@@ -118,6 +118,8 @@ pub enum RippleContract {
     /// Distributor gets the ability to configure and customize the generation of
     /// the Session information based on their policies. Used by [crate::api::session::AccountSession]
     Session(SessionAdjective),
+    /// Provides platform token for a given device.
+    PlatformSessionToken,
```
Comment (satlead) [inline core/sdk/src/framework/ripple_contract.rs:122]: Please use SessionAdjective::Platform for this contract

File: core/sdk/src/api/distributor/distributor_platform.rs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    fn get_from_payload(payload: ExtnPayload) -> Option<PlatformTokenRequest> {
+        if let ExtnPayload::Request(ExtnRequest::PlatformToken(r)) = payload {
+            return Some(r);
+        }
+
+        None
+    }
+
+    fn contract() -> RippleContract {
+        RippleContract::PlatformSessionToken
```
Comment (satlead) [inline core/sdk/src/api/distributor/distributor_platform.rs:56]:
```suggestion
          RippleContract::Session(SessionAdjective::Platform),
```

**Review by satlead — APPROVED** [review]
Wait for a QA approval as well before merging

**Review by satlead — APPROVED** [review]
lgtm

**Review by Nv-Delle1416 — APPROVED** [review]
QA has been completed on these changes - Approved

**Pull request merged**